ssh/tailssh/tailssh.go

```diff
@@ -150,7 +150,7 @@ func (srv *server) handleSSH(s ssh.Session) {
 	}
 	action, localUser, ok := evalSSHPolicy(pol, sctx)
 	if ok && action.Message != "" {
-		io.WriteString(s, action.Message)
+		io.WriteString(s.Stderr(), strings.Replace(action.Message, "\n", "\n\r", -1))
```

Comment:
fine for now, but we might end up adding a bunch of carriage returns when the input is `\r\n` already. consider pulling it out into a different func and adding a comment.

Reply:
tweaked to \r\n for now but until we have half of a plan here, leaving as-is instead of pretending we do